packages/smarthr-ui/src/components/ErrorScreen/stories/VRTErrorScreen.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+import { ErrorScreen } from '../ErrorScreen'
+
+import type { Meta, StoryObj } from '@storybook/react'
+
+export default {
+  title: 'Page Templates（ページテンプレート）/ErrorScreen',
```

Comment:
```suggestion
  title: 'Page Templates（ページテンプレート）/ErrorScreen/VRT',
```

他に合わせると `/VRT` なんですが、Chromatic ではディレクトリ切らないほうが見やすいですね😇

Reply:
あ、普通にミスなので修正します！

確かに Chromatic 上でどのコンポーネントなのか全然わからないですね…。
すべてが終わったら一括で変えちゃうのが良さそう。